telemetry/url.go

```diff
@@ -0,0 +1,15 @@
+package telemetry
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/trace"
+)
+
+func URLForTrace(ctx context.Context) (string, bool) {
```

Comment:
Fun little idea about this - this function could imply the existence of a `URLForSpan` function :tada:

On a failure, we could print out the URL of a failed span (or even use [OSC 8](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda) to display it inline!